actions/templates.go

```diff
@@ -137,37 +132,35 @@ func PostTemplate(c *gin.Context) {
 		return
 	}
 
-	store, err := templates.NewSesTemplateStore(keys.AccessKey, keys.SecretKey, keys.Region)
+	// todo ask about text part and html part should we change now post params ?
+	templateInput := &entities.Template{
+		UserID:      u.ID,
+		Name:        body.Name,
+		HTMLPart:    body.Content,
+		TextPart:    body.Content,
+		SubjectPart: body.Subject,
+	}
+
+	// todo discuss about validation
+	// create new template and try to parse html part
+	_, err := template.New("new").Parse(templateInput.HTMLPart)
```

Comment:
use the template name when you create the new template instead of new

Reply:
also you need to parse both text and html parts i think

Reply:
I think the validation should be done inside the `service`, instead of the handler. We will use https://github.com/cbroglie/mustache for our templates instead of the std lib `template`, because it's simpler and does not have any for loops and conditionals.

The validation can be done like so:

`tmpl, err := mustache.ParseString(template.HTMLPart)`

and handle the `err` correctly.